etc/examples/paths.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# these can be set by folks running the examples/Makefile.testing to use a different binaries than in the path
+ECHO=${ECHO:-echo}
```

Comment:
What platform-specific behavior is being use with echo that requires this? All I see are the `-e` and `-n` flags being employed, both of which are supported by macOS.

Reply:
-e isn't supported on macOS.  You need to support gnubin utils to get it.

Reply:
`man echo` on macOS implies otherwise; it gives the same description for `-e` as the ubuntu variant

Reply:
Hmmm... Catalina shows just the -n flag supported, and using "-e" with ANSI codes doesn't do anything.  The gnu version works correctly.   What macOS are you on?

Reply:
Catalina

Reply:
Interesting.  Different results on my Catalina, where I'm using bash. The built-in echo on zsh understands the -e flag, but bash uses /bin/echo, which doesn't.  Which supports my case: even different shells will give different results, so giving some degree of control over this to the invoker of the makefile makes sense.